### PR TITLE
Initial Changes to To do Items

### PR DESCRIPTION
**Pull request by NShilu**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

**Review by JackMeadDev — CHANGES_REQUESTED** [review]
Nice work, I was easily able to run the app and see the to-dos so the first part is done, but it doesn't look like you've completed the second part which is to also add functionality for creating new to-dos; maybe something you forgot to push? Feel free to reach out if you're having issues 🙂

**Comment by NShilu** [comment]
Hi Jack and Alex,

I believe I followed the instructions and created the Pull request for a new branch ‘exercise-1’, I am unsure what and where I missed ?

Shall I arrange a quick session with one of you to address this please

Thanks
Nirav


Nirav Shilu
Professional Services Consultant

retaininternational part of the Capita Group

***@***.***
Office +44 (0) 20 7538 4774
Support +44 (0) 20 7538 9237
Email: ***@***.******@***.***>
Web: www.retaininternational.com<http://www.retaininternational.com/>

File: todo_app/app.py [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+
+@app.route('/addItem', methods=['POST'])
+
+def index_addItem():
+
+    title=request.form.get('title')
+
+    add_item(title)
+
+    return redirect('/')
```
Comment (Jonesey13) [inline todo_app/app.py]: To keep consistent spacing
```suggestion
    title=request.form.get('title')
    add_item(title)
    return redirect('/')
```

File: todo_app/app.py [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template("index.html", items = items)
+
+@app.route('/addItem', methods=['POST'])
+
+def index_addItem():
```
Comment (Jonesey13) [inline todo_app/app.py]: Convention in Python is for method names to be `lower_snake_case` (see PEP8: https://www.python.org/dev/peps/pep-0008/)
```suggestion
def add_item():
```

File: todo_app/app.py [hunk 3]
```diff
@@ -1,11 +1,25 @@
...
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template("index.html", items = items)
+
+@app.route('/addItem', methods=['POST'])
+
```
Comment (Jonesey13) [inline todo_app/app.py:17]:
```suggestion
@app.route('/addItem', methods=['POST'])
```

**Review by Jonesey13 — APPROVED** [review]
Looks good to me. Left a few comments but feel free to merge in when ready 👍

**Pull request merged**